combo_cases_and_deaths/delphi_combo_cases_and_deaths/run.py

```diff
@@ -83,15 +105,12 @@ def sensor_signal(metric, sensor, smoother):
         sensor_name = "_".join([smoother, sensor])
     else:
         sensor_name = sensor
-    signal = "_".join([metric, sensor_name])
-    return sensor_name, signal
-
-def run_module():
-    """Produce a combined cases and deaths signal using data from JHU and USA Facts"""
-    variants = [tuple((metric, geo_res)+sensor_signal(metric, sensor, smoother))
-                for (metric, geo_res, sensor, smoother) in
-                product(METRICS, GEO_RESOLUTIONS, SENSORS, SMOOTH_TYPES)]
+    return sensor_name, "_".join([metric, sensor_name])
 
+def configure(variants):
```

Comment:
Is the `variants` argument used here at all?

Reply:
[Yes](https://github.com/cmu-delphi/covidcast-indicators/blob/9e46bc28248295a1b5c2db9a3cb9bd54291f63fe/combo_cases_and_deaths/delphi_combo_cases_and_deaths/run.py#L125)

Reply:
Ah, I missed a code fold.